challenges/objects-arrays.js

```diff
@@ -59,11 +85,18 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+graduates.forEach(function(currentValue){
+  contactInfo.push(`${currentValue.first_name} ${currentValue.email}`)
+});
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+let uni = graduates.filter(function(currentValue){
+  if (currentValue.university.includes("Uni") === true) {
```

Comment:
dont actually have to use the equality check here, you can just say `if (currentValue.university.includes("Uni"))` because .includes() will return a boolean, and thats what our if is checking for